docs/frameworks/additional-requests.rst

```diff
@@ -0,0 +1,228 @@
+.. _framework-additional-requests:
+
+==============================
+Supporting Additional Requests
+==============================
+
+To support :ref:`additional requests <advanced-requests>`, your framework must
+provide the request download implementation of :class:`~.HttpClient`.
+
+.. _advanced-downloader-impl:
```

Comment:
Is it a right time to clean up ref labels?

Reply:
I have no strong opinion either way. Since they are not user-visible, I tend not to worry about them until I need to touch related code.